openslides/core/models.py

```diff
@@ -294,6 +297,76 @@ def __str__(self):
         return 'Message {}'.format(self.timestamp)
 
 
+class Message(RESTModelMixin, models.Model):
+    """
+    Model for messages.
+    """
+    access_permissions = MessageAccessPermissions()
+
+    message = models.TextField(blank=True)
+
+    identify = models.BooleanField(default=False)
```

Comment:
It's just a boolean representing if this message is an identifier message or not.